js_modules/dagit/packages/core/src/scripts/generateGraphQLTypes.ts

```diff
@@ -7,20 +7,17 @@ console.log('Downloading schema...');
 
 const TARGET_FILE = './src/graphql/schema.graphql';
 
-// https://github.com/dagster-io/dagster/issues/2623
+// Write schema.graphql in the SDL format, without descriptions
 const result = execSync(
   `dagster-graphql --ephemeral-instance --empty-workspace -t '${getIntrospectionQuery({
     descriptions: false,
   })}'`,
   {cwd: '../../../../examples/docs_snippets/'},
```

Comment:
this cwd is confusing, is this related to the github issue that is commented out here? Should we leave that in?